jscomp/others/jsxDOM.ml

```diff
@@ -0,0 +1,2087 @@
+  ariaReadonly: bool
+  [@ns.optional] [@bs.as "aria-readonly"];
+  ariaRequired: bool
+  [@ns.optional] [@bs.as "aria-required"];
+  ariaSelected: bool
+  [@ns.optional] [@bs.as "aria-selected"];
+  ariaSort: string
+  [@ns.optional] [@bs.as "aria-sort"];
+  ariaValuemax: float
+  [@ns.optional] [@bs.as "aria-valuemax"];
+  ariaValuemin: float
+  [@ns.optional] [@bs.as "aria-valuemin"];
+  ariaValuenow: float
+  [@ns.optional] [@bs.as "aria-valuenow"];
+  ariaValuetext: string
+  [@ns.optional] [@bs.as "aria-valuetext"];
+  (* Live Region Attributes *)
+  ariaAtomic: bool
+  [@ns.optional] [@bs.as "aria-atomic"];
+  ariaBusy: bool
+  [@ns.optional] [@bs.as "aria-busy"];
+  (* [@ns.optional] [@bs.as "aria-live"] ariaLive: off|polite|assertive|rude *)
+  ariaRelevant: string
+  [@ns.optional] [@bs.as "aria-relevant"];
+  (* Drag-and-Drop Attributes *)
+  (* [@ns.optional] [@bs.as "aria-dropeffect"] ariaDropeffect: copy|move|link|execute|popup|none *)
+  ariaGrabbed: bool
+  [@ns.optional] [@bs.as "aria-grabbed"];
+  (* Relationship Attributes *)
+  ariaActivedescendant: string
+  [@ns.optional] [@bs.as "aria-activedescendant"];
+  ariaColcount: int
+  [@ns.optional] [@bs.as "aria-colcount"];
+  ariaColindex: int
+  [@ns.optional] [@bs.as "aria-colindex"];
+  ariaColspan: int
+  [@ns.optional] [@bs.as "aria-colspan"];
+  ariaControls: string
+  [@ns.optional] [@bs.as "aria-controls"];
+  ariaDescribedby: string
+  [@ns.optional] [@bs.as "aria-describedby"];
+  ariaErrormessage: string
+  [@ns.optional] [@bs.as "aria-errormessage"];
+  ariaFlowto: string
+  [@ns.optional] [@bs.as "aria-flowto"];
+  ariaLabelledby: string
+  [@ns.optional] [@bs.as "aria-labelledby"];
+  ariaOwns: string
+  [@ns.optional] [@bs.as "aria-owns"];
+  ariaPosinset: int
+  [@ns.optional] [@bs.as "aria-posinset"];
+  ariaRowcount: int
+  [@ns.optional] [@bs.as "aria-rowcount"];
+  ariaRowindex: int
+  [@ns.optional] [@bs.as "aria-rowindex"];
+  ariaRowspan: int
+  [@ns.optional] [@bs.as "aria-rowspan"];
+  ariaSetsize: int
+  [@ns.optional] [@bs.as "aria-setsize"];
+  (* react textarea/input *)
+  defaultChecked: bool
+  [@ns.optional];
+  defaultValue: string
+  [@ns.optional];
+  (* global html attributes *)
+  accessKey: string
+  [@ns.optional];
+  className: string (* substitute for "class" *)
+  [@ns.optional];
+  contentEditable: bool
+  [@ns.optional];
+  contextMenu: string
+  [@ns.optional];
+  dir: string (* "ltr", "rtl" or "auto" *)
+  [@ns.optional];
+  draggable: bool
+  [@ns.optional];
+  hidden: bool
+  [@ns.optional];
+  id: string
+  [@ns.optional];
+  lang: string
+  [@ns.optional];
+  role: string (* ARIA role *)
+  [@ns.optional];
+  style: style
+  [@ns.optional];
+  spellCheck: bool
+  [@ns.optional];
+  tabIndex: int
+  [@ns.optional];
+  title: string
+  [@ns.optional];
+  (* html5 microdata *)
+  itemID: string
+  [@ns.optional];
+  itemProp: string
+  [@ns.optional];
+  itemRef: string
```

Comment:
Is this file still used?

Reply:
Frankly, not sure. As an initial step, I tried to move all properties first. I think we can trim it down later.

Reply:
Can you expand the answer?
With the current PR to rescript-react, and testing on some project, is this file used or not?

Reply:
Ah, I misunderstood your question, I thought you ask only whether `itemProps` and `itemRef` properties are used or not. Yes, `JsxDOM` is being used in rescript-react for new jsx mode. https://github.com/rescript-lang/rescript-react/blob/2d3a7dfbf1c30f0b68359730ec0ad486be04727f/src/ReactDOM.res#L2106